### ISSUE TITLE
Documentation of the AndInThreads class needs to be updated

### ISSUE THREAD

**Issue by driver733** [yegor256/cactoos#688]
This is an example from class javadoc in the AndInThreads class.
```
  new AndInThreads(
            new IterableOf("Mary", "John", "William", "Napkin"), 
            name -> System.out.printf("The name: %s\n", name)
   ).value();
```

This code does not compile and shows a warning.

Compiler error:
`Cannot resolve method 'printf(java.lang.String, <lambda parameter>)'`
Warning:
`Unchecked call to 'IterableOf(X...)' as a member or raw type 'org.cactoos.iterable.IterableOf'`

https://github.com/yegor256/cactoos/pull/687#discussion_r169408286
https://github.com/yegor256/cactoos/pull/687#discussion_r169345396

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@driver733 fixed in 29819e4 thanks!
